.github/ISSUE_TEMPLATE/docs_request.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Docs Request for an Update or Improvement'
+description: A request to update or improve euphony-hub-collector documentation
+title: 'Docs: '
```

Comment:
This also should be updated due to consistency. Please refer other templates in euphony-hub-collector :)

Reply:
Could you please provide more details? I don't get it properly.

Reply:
you can check title of bug-report:
https://github.com/euphony-io/euphony-hub-collector/blob/master/.github/ISSUE_TEMPLATE/bug-report.yaml